### ISSUE TITLE
Test the compilation of MET on a Mac prior to met-7.1

### ISSUE THREAD

**Issue by dwfncar** [dtcenter/MET#981]
This task is the result of a met_help ticket from Dave Ahijevyc:
<br/>
<a href="https://rt.rap.ucar.edu/rt/Ticket/Display.html?id=84433">https://rt.rap.ucar.edu/rt/Ticket/Display.html?id=84433</a>
<br/>

<br/>
Test compiling on a Mac to resolve this issues compiling MET.
<br/>

<br/>
++++++++++++++++++++++++++++++++++++++++
<br/>

<br/>
Had to add <limits.h> to ./src/basic/vx_config/config_util.cc
<br/>

<br/>
to get past make error.
<br/>

<br/>
As told in
<br/>
<a href="http://mailman.ucar.edu/pipermail/met_help/2017-June/002933.html">http://mailman.ucar.edu/pipermail/met_help/2017-June/002933.html</a>
<br/>

<br/>
Replace single quotes with double quotes in &#39;\0&#39;).
<br/>
In other words replace   &#39;\0&#39;) with "\0") in ./src/libcode/vx_
<br/>
statistics/compute_ci.cc
<br/>
src/tools/core/stat_analysis/stat_analysis.cc
<br/>
and src/tools/other/pb2nc/pb2nc.cc
<i>[MET-981] created by johnhg</i>
